CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Line wrap the file at 100 chars.                                              Th
 - Enable isolation of the Electron renderer process to protect against potentially malicious third
   party dependencies.
 - Add 51820 to list of WireGuard ports in app settings.
+- Add support for Apple Silicon
```

Comment:
Since we won't release Apple Silicon builds until the remaining issues have been resolved we'll have to wait with including it in the changelog.

Reply:
Done